zio-crypto/shared/src/main/scala/zio/crypto/encryption/HybridEncryption.scala

```diff
@@ -78,8 +79,8 @@ object HybridEncryption {
    * @param key: The public key to use to encrypt the message.
    * @return the `CipherText` generated from encrypting `plainText` with `key`.
    */
-  def encrypt(plainText: Chunk[Byte], key: PublicKey): RIO[Has[HybridEncryption], CipherText[Chunk[Byte]]] =
-    ZIO.accessM(_.get.encrypt(plainText, key))
+  def encrypt(plainText: Chunk[Byte], key: PublicKey): RIO[HybridEncryption, CipherText[Chunk[Byte]]] =
+    ZIO.environmentWithZIO(_.get.encrypt(plainText, key))
```

Comment:
This could be `ZIO.serviceWithZIO(_.encrypt(plainText, key))`

Reply:
The scalafix from the migration did all of these. I noticed it changed `.accessM` to `environmentWithZIO`, but it didn't updated anything that was `.access`. I noticed this behavior with a work project I was testing migrations on for an earlier RC. Is this something that might need to be updated in the scalafix rule?

Either way, it probably makes more sense to make them `. serviceWithZIO` - I'll update them.

Reply:
🤦 Didn't notice the `.get` right away.